lib/values/string_constants.dart

```diff
@@ -1,27 +1,32 @@
 class AppStrings {
-  static const welcome = 'Welcome';
+  const AppStrings._();
+
+  static const String welcome = 'Welcome';
 
   //Signup strings
-  static const signIn = 'Sign In';
-  static const signUp = 'Sign Up';
-  static const hintName = 'Name';
-  static const hintEmail = 'Email,phone number';
-  static const hintPassword = 'Password';
-  static const rememberMe = 'Remember me';
-  static const haveAccount = 'Already have an account?';
+  static const String signIn = 'Sign In';
+  static const String signUp = 'Sign Up';
+  static const String hintName = 'Name';
+  static const String hintEmail = 'Email,phone number';
+  static const String hintPassword = 'Password';
+  static const String rememberMe = 'Remember me';
+  static const String haveAccount = 'Already have an account?';
 
   //Sign in strings
-  static const hintUser = 'Phone, email, username';
-  static const forgotPassword = 'Forgot Password';
-  static const dontHaveAccount = "Don't have an account?";
-  static const register = 'Register now';
-  static const forgotPasswordValue = 'Toh rehndo phir 😜';
+  static const String hintUser = 'Phone, email, username';
+  static const String forgotPassword = 'Forgot Password';
+  static const String dontHaveAccount = "Don't have an account?";
+  static const String register = 'Register now';
+  static const String forgotPasswordValue = 'Toh rehndo phir 😜';
 
   //Validation errors
-  static const emptyError = 'Please enter some text';
-  static const nameError = 'Please enter a valid name';
-  static const multipleError =
+  static const String emptyError = 'Please enter some text';
+  static const String nameError = 'Please enter a valid name';
+  static const String multipleError =
       'Input does not match any of the allowed formats';
-  static const passwordError =
+  static const String passwordError =
       'Password must be at least 8 characters long and contain at least one letter and one number';
+
+  //Assets
+  static const String arrowAsset = 'assets/images/arrow.png';
```

Comment:
Please maintain asset constants in a separate file.

Reply:
Resolved